src/vs/workbench/browser/layout.ts

```diff
@@ -214,6 +216,10 @@ export abstract class Layout extends Disposable implements IWorkbenchLayoutServi
 			panelToRestore: undefined as string | undefined
 		},
 
+		banner: {
```

Comment:
we can remove this for now, forgot to clean it up but its not really in use since we never set it to true